style.css

```diff
@@ -45,3 +67,33 @@ body {
   border-color: #ff6b6b;
   color: #ff6b6b;
 }
+
+/* removes the possibility of reselecting the divs during animations */
```

Comment:
I don't like this solution (although it is smart) - we should stick to using JS for logic, and ignore the pointer events in the listener function when appropriate conditions occur. You moved part of the logic to the styles, which make it very hard to test and not obvious for fellow developers (who will look at the JS in search for program logic)